supernovacontroller/sequential/i3c.py

```diff
@@ -951,6 +959,7 @@ def ccc_unicast_enec(self, target_address):
 
         Args:
         target_address: The address of the target device on the I3C bus to which the ENEC command is directed.
+        events: A list containing events to be enabled.
```

Comment:
Please, mention that the type of element of the list belongs to the enum ENEC: ENEC.ENINT, ENEC.ENCR, ENEC.ENHJ. For instance [ENEC.ENINT, ENEC.ENHJ]